content/en/xtc/basics/045-organizations.md

```diff
@@ -51,3 +51,50 @@ In _Configuration_, you may edit the name, short name and description of your or
 
 {{% permission role="organization administrator" %}}
 
+### Mandatory Login Requirements
+
+{{% permission role="organization administrator" %}}
+
+Organization admins may impose mandatory login requirements regarding on users of their organization. They may require users to be authenticated using specific [login providers]({{< relref "040-user-accounts/#connecting-existing-xtc-accounts-to-an-external-login-provider" >}}) or [2FA]({{< relref "040-user-accounts/#two-factor-authentication-2fa" >}}).
+
+{{% note notitle %}}
+The configuration _DOES NOT_ force users to actually configure the required login procedure to be able to log into XTC. It merely prevents them from accessing the organization. Users will still be able to see the organization in their dashboard and organization dropdown, but will see an appropriate error screen informing them about the need to comply with the login requirement when trying to access the organization details. Oganization admins _SHOULD_ inform users via appropriate means to ensure that they are up-to-date on any login changes.
```

Comment:
I'd suggest bold text for emphasis instead of italic caps which look kinda ugly imho.